### PR TITLE
UF-5679: Implemented decision task worker

### PR DESCRIPTION
**Pull request by MickeWallner**

### CODE REVIEW

File: src/main/java/se/sundsvall/parkingpermit/businesslogic/util/BusinessRulesUtil.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+			return null;
+		}
+
+		return transformed.concat(".");
+	}
+
+	private static String transformToString(List<String> strings) {
+		String concatenated = String.join(", ", ofNullable(strings).orElse(emptyList()));
+
+		return capitalize(concatenated.replaceAll(REGEXP_LAST_COMMA, "$1 och $3"));
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/parkingpermit/businesslogic/util/BusinessRulesUtil.java:68]: Do you really need an `"och"` before the last element? Is this a requirement?

If not:
`return Optional.ofNullable(list).orElse(emptyList()).stream().collect(Collectors.joining(", "));`

Comment (MickeWallner) [reply src/main/java/se/sundsvall/parkingpermit/businesslogic/util/BusinessRulesUtil.java:68]: I don't know if it's a requirement but in the old service the description is composed like this and I think it's quite nice. An example:
Rekommenderat beslut är bevilja. Den sökande är helt rullstolsburen, funktionsnedsättningens varaktighet är 6 månader eller längre och den sökande har inga aktiva parkeringstillstånd.

Comment (foobar-1337) [reply src/main/java/se/sundsvall/parkingpermit/businesslogic/util/BusinessRulesUtil.java:68]: Yes, it's nice. I don't disagree.
But you will have to do something at the security issue. That's why I commented this part of the code.

Comment (MickeWallner) [reply src/main/java/se/sundsvall/parkingpermit/businesslogic/util/BusinessRulesUtil.java:68]: I suspected that :)

**Review by foobar-1337 — APPROVED** [review]

**Pull request merged**